BangazonAPI/TestBangazonAPI/DatabaseFixture.cs

```diff
@@ -91,7 +122,9 @@ public void Dispose()
                 conn.Open();
                 using (SqlCommand cmd = conn.CreateCommand())
                 {
-                    cmd.CommandText = @$"DELETE FROM PaymentType WHERE AcctNumber='test'";
+                    cmd.CommandText = @$"DELETE FROM ProductType WHERE Name='Test Product Type' ";
+                    cmd.ExecuteNonQuery();
```

Comment:
You only need to execute your sql command once-- you can just keep adding to the command text (either by using a `+=` or by just adding to the string itself) and then execute at the end. This works, but it's making your test do a little more work than they need to. This can be a later refactor, it doesn't have to be on this PR, but I wanted to point it out so that you can clean it up in the future. Eventually this would slow your tests down quite a bit.